pkg/jx/cmd/install.go

```diff
@@ -356,6 +360,8 @@ func (options *InstallOptions) addInstallFlags(cmd *cobra.Command, includesInit
 	cmd.Flags().BoolVarP(&flags.Kaniko, "kaniko", "", false, "Use Kaniko for building docker images")
 	cmd.Flags().BoolVarP(&flags.NextGeneration, "ng", "", false, "Use the Next Generation Jenkins X features like Prow, Tekton, No Tiller, Vault, Dev GitOps")
 	cmd.Flags().BoolVarP(&flags.StaticJenkins, "static-jenkins", "", false, "Install a static Jenkins master to use as the pipeline engine. Note this functionality is deprecated in favour of running serverless Tekton builds")
+	cmd.Flags().BoolVarP(&flags.LongTermStorage, "long-term-storage", "", false, "Enable the Long Term Storage option to save logs and other assets into a GCS bucket")
```

Comment:
The way it is worded now, it is a GKE option only? In the case of Vault we use something like:

> Sets up a Hashicorp Vault for storing secrets during installation (supported only for GKE)

Maybe we should align how we present this in the help string.